src/Components/Projects/Form/index.js

```diff
@@ -0,0 +1,321 @@
+import { useEffect, useState } from 'react';
+import styles from './form.module.css';
+
+const Form = () => {
+  const [projectAdd, setProjectAdd] = useState({
+    name: '',
+    clientName: '',
+    description: '',
+    startDate: '',
+    endDate: '',
+    employees: []
+  });
+  const [employeeList, setEmployeeList] = useState([]);
+  const [employeeProject, setEmployeeProject] = useState({
+    rate: null,
+    role: '',
+    employee: ''
+  });
+  const [errorMessage, setErrorMessage] = useState();
+  const [form, setForm] = useState(true);
+
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_API_URL}/employees`)
+      .then((response) => response.json())
+      .then((response) => {
+        setEmployeeList(response.data);
+      });
+  }, []);
+
+  const formatDate = (date) => {
+    const dateIso = date.substr(0, 10);
+    return dateIso;
+  };
+
+  useEffect(async () => {
+    const params = new URLSearchParams(window.location.search);
+    const id = params.get('id');
+    if (id) {
+      try {
+        const response = await fetch(`${process.env.REACT_APP_API_URL}/projects/${id}`, {
+          method: 'GET'
+        });
+        const project = await response.json();
+        const employeeList = [];
+        project.data.employees.map((item) => {
+          delete item.employee.name;
+          delete item.employee.email;
+          delete item.employee.lastName;
+          delete item.employee.password;
+          delete item.employee.phone;
+          let employeeId = item.employee._id;
+          item.employee = employeeId;
+          employeeList.push(item);
+        });
+        setProjectAdd({
+          name: project.data.name,
+          clientName: project.data.clientName,
+          description: project.data.description,
+          startDate: project.data.startDate,
+          endDate: project.data.endDate,
+          employees: employeeList
+        });
+        setForm(false);
+      } catch (error) {
+        console.log(error);
+      }
+    } else {
+      return null;
+    }
+  }, []);
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+  };
+
+  const deleteEmployees = (id) => {
+    setProjectAdd({
+      ...projectAdd,
+      employees: projectAdd.employees.filter((employee) => employee.employee !== id)
+    });
+  };
+
+  const formProjects = async (projectAdd) => {
```

Comment:
```suggestion
  const onSubmit = async () => {
```
It's not necessary to use a param here because we are going to use the `projectAdd` that comes from the useState.